beacon-chain/blockchain/service.go

```diff
@@ -137,6 +137,9 @@ func (s *Service) Start() {
 		}
 	}
 
+	// Make sure that attestation processor is subscribed and ready for state initializing event.
+	attestationProcessorSubscribed := make(chan struct{}, 1)
```

Comment:
Do we need to make sure it is unbuffered so that it will block?

Reply:
Well,  it will block on reader's side (which is what we need). And on writer, we can just write w/o blocking.